src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -1657,13 +1645,76 @@ class CaseExpressionSegment(BaseSegment):
 )
 
 
+@ansi_dialect.segment()
+class EqualsSegment(BaseSegment):
+    """Equals operator."""
+
+    type = "comparison_operator"
+    name = "equals"
+    match_grammar = Ref("RawEqualsSegment")
+
+
+@ansi_dialect.segment()
+class GreaterThanSegment(BaseSegment):
+    """Greater than operator."""
+
+    type = "comparison_operator"
+    name = "greater_than"
+    match_grammar = Ref("RawGreaterThanSegment")
+
+
+@ansi_dialect.segment()
+class LessThanSegment(BaseSegment):
+    """Less than operator."""
+
+    type = "comparison_operator"
+    name = "less_than"
+    match_grammar = Ref("RawLessThanSegment")
+
+
+@ansi_dialect.segment()
+class GreaterThanOrEqualToSegment(BaseSegment):
+    """Greater than or equal to operator."""
+
+    type = "comparison_operator"
+    name = "greater_than_equal_to"
+    match_grammar = Sequence(
+        Ref("RawGreaterThanSegment"), Ref("RawEqualsSegment"), allow_gaps=False
+    )
+
```

Comment:
You can see the actual comparison operators are built from the raw_comparison_operators and ensures consistent AST for all comparison operators.